bridges/VkBridge.php

```diff
@@ -109,19 +109,9 @@ public function getContents()
 	{
 		ini_set('user-agent', 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:53.0) Gecko/20100101 Firefox/53.0');
 
-		$opts = array(
-			'http' => array(
-				'method' => "GET",
-				'user_agent' => ini_get('user_agent'),
-				'accept_encoding' => 'gzip',
-				'header' => "Accept-language: en\r\n 
-					Cookie: remixlang=3\r\n"
-			)
-		);
-
-		$context = stream_context_create($opts);
+		$header = array("Accept-language: en\r\nCookie: remixlang=3\r\n");
```

Comment:
Can be rewritten to 
```php
array("Accept-language: en", "Cookie: remixlang=3");
```
http://php.net/manual/en/function.curl-setopt.php
Search for CURLOPT_HTTPHEADER